docs/.gitlintrc.example

```diff
@@ -0,0 +1,10 @@
+preset: default # optional, defaults to "default" preset
+rules:
+	subject-no-trailing-whitespace: 
+		- never # rule condition, is never used
```

Comment:
Is that the first item in the array must be `never` or `always`? It seems confusing to describe a rule as `never`/`always`. Maybe use `on`/`off`?

Reply:
This was the convention that eslint and commitlint uses

Reply:
ESLint uses `error`, `warn`, and `off` levels. I just checked documentation of commitlint, and it seems to use a rather exotic semantics of rule configuration: `never` does not turn off the rule, but rather 'invert' the rule. A `subject-empty` rule configured with `never` would pass if the subject is not empty. Do you think we should follow their configuration?

Reply:
Okay, let's stick with `on`/`off` for enabling/disabling rules then